dotcom-rendering/src/web/components/TrendingTopics.tsx

```diff
@@ -8,10 +8,12 @@ type Props = {
 };
 
 const linkStyle = css`
+	font-family: 'Guardian Text Sans Web', 'Helvetica Neue', Helvetica, Arial,
+		'Lucida Grande', sans-serif;
 	text-decoration: none;
-	font-size: 16px;
+	font-size: 14px;
 	top: 0;
-	line-height: 16px;
+	line-height: 14px;
```

Comment:
As @jamesgorrie says, should this be:

```ts
${textSans.xsmall({ lineHeight: 'tight' })}
```

where `textSans` comes from `source-foundations`? `xsmall` corresponds to `14px`. This will also give a `line-height` of `~16px`, as "tight" is 115% of font-size. This is the minimum supported for text sans I think?